packages/cli/test/commands/deploy.test.ts

```diff
@@ -12,6 +12,7 @@ import * as packageManagerImpl from '../../src/services/package-manager/live.imp
 import * as configService from '../../src/services/config-service'
 import * as projectChecker from '../../src/services/project-checker'
 import { makeTestPackageManager } from '../services/package-manager/test.impl'
+import { root } from '../configRoot'
```

Comment:
I don't fully understand how this `configRoot` file and manually setting the path `/someDir` in the tests works. Could you explain why you needed to do this and how the solution works?

This is not your fault, but the CLI tests are currently using way too much "real stuff" to call them "unit tests". Using actual environment variables and depending on certain files in the filesystem is a design flaw. It makes tests slower and less reliable, and the root reason for the current issue with the CLI unit tests likely comes from this faulty design... If this is a patch to workaround this problematic design, that's okay for now, because we want to add support for Node 18 ASAP, but we should probably consider re-designing the CLI unit tests to make sure they perform no actual syscalls nor access real environment variables... We can leave this for another future PR in any case :-D

Reply:
I'm not sure why, but at least from the upgrade node branch, some tests were failing due to none command generated with oclif being found. Debugging the test I found out that the configuration was not being loaded so the only way I found (that is also used in the oclif repo tests) was creating a const that joins `__dirname` with the parent folder and use this variable to imperatively load the config. So I created the configRoot helper to not init the const in each file.